Assets/HoloToolkit/UX/Scripts/Tooltip/TooltipAttachPointTypeEnum.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HoloToolkit.UX.ToolTips
+{
+    /// <summary>
+    /// Used to find a pivot point that is closest to the 
+    /// anchor. This ensures a natural-looking attachment where the connector line
+    /// meets the label.
+    /// </summary>
+    public enum ToolTipAttachPointType
+    {
+        // Specific options
+        // These double as array positions
+        BotMiddle = 0,
+        TopMiddle = 1,
+        RightMiddle = 2,
+        LeftMiddle = 3,
+        BotRightCorner = 4,
+        BotLeftCorner = 5,
+        TopRightCorner = 6,
+        TopLeftCorner = 7,
+        // Automatic options
+        Center,
```

Comment:
If ClosestCorner becomes 0, it will be important to also update the dev branch. otherwise apps will get an unexpected setting change when migrating

Reply:
Actually, ClosestCorner cannot be zero because the first eight enums are fixed as indices into an array.
ClosestCorner is one of a collection of runtime-computed enum options so it has to be defined after the initial values of 0-7